lib/puppet/type/cisco_vrf_af.rb

```diff
@@ -192,6 +226,29 @@ def insync?(is)
     end
   end # route_target_import_evpn
 
+  newproperty(:route_target_import_stitching, array_matching: :all) do
+    desc '(stitching only) Set the route-target import stitching communities. '\
+         'Valid values are an Array, a space-separated String of extended '\
+         "communities, or the keyword 'default'."
+
+    match_error = 'must be specified in AS:nn or IPv4:nn notation'
+    validate do |community|
+      community.split.each do |value|
+        fail "Confederation peer value '#{value}' #{match_error}" unless
```

Comment:
I actually did not find any documentation on this. From everything I tried, it works as route_target_import, except they are the stitching RT.